src/configure/model/models.ts

```diff
@@ -52,6 +52,29 @@ export class Organization {
     isMSAOrg: boolean;
 }
 
+export class GitHubOrganization {
+    login: string;
+    id : number;
+    node_id: string;
+    url: string;
+    repos_url: string;
+    events_url: string;
+    hooks_url: string;
+    issues_url: string;
+    members_url: string;
+    public_members: string;
+    public_members_url: string;
+    avatar_url: string;
+    description: string;
+}
+
+export class GitHubRepo{
+    id: string;
+    name: string;
+    orgName: string;
+    html_url: string;
```

Comment:
is this required?

Reply:
Yes, for receiving data from API call used for creating a new GitHub repository.